client/devdocs/design/index.jsx

```diff
@@ -101,6 +102,7 @@ export default React.createClass( {
 					<InfoPopover />
 					<InputChrono />
 					<Notices />
+					<NoticesGlobal />
```

Comment:
What would you call it? I wanted to call it `GlobalNotices` but I also wanted it to appear under notices, and someone decided to alphabetize the list :)